### ISSUE TITLE
myproxy RPMs from GlobusToolkit need to depend on (exact version of) myproxy-libs

### ISSUE THREAD

**Issue by alaniwi** [globus/globus-toolkit#85]
Per issue described at https://github.com/ESGF/esgf-installer/issues/161 (see the main long paragraph).

**Comment by alaniwi** [comment]
Correction: it seems that the dependency on `myproxy-libs` has been lost altogether. This is in `myproxy-6.1.21-2.el6+gt6.x86_64`.

The older version on our system (`6.1.18-1.el6+gt6`) did in fact have an exact version dependency (`myproxy-libs = 6.1.18-1.el6+gt6`), although of course this did not prevent a `yum update myproxy` from updating `myproxy` to 6.1.21 while leaving `myproxy-libs` unchanged.

Before:

```
# rpm -q myproxy
myproxy-6.1.18-1.el6+gt6.x86_64
# rpm -qR myproxy | grep myproxy-libs
myproxy-libs = 6.1.18-1.el6+gt6
#
```

After:

```
# rpm -q myproxy
myproxy-6.1.21-2.el6+gt6.x86_64
# rpm -qR myproxy | grep myproxy-libs
#
```

So essentially, whatever you were doing previously was correct, but needs to be put back again.

**Comment by bester** [comment]
Fixed in myproxy RPM version 6.1.25

**Comment by alaniwi** [comment]
Thank you.
